content/contact.md

```diff
@@ -19,19 +19,19 @@ The CHipCie takes care of the programming contest within Delft and consist of th
 From left to right:
 * Nadyne Aretz, _Chairwoman_
 * Laura Pircalaboiu, _Treasurer_
-* Raoul Bruens, _Commisioner of Logistics & Activities_
-* Brian Witmer, _Commisioner of Recruitment & Promotion_
-* Maarten Sijm, _Commisioner of Jury Affairs_
+* Raoul Bruens, _Commissioner  of Logistics & Activities_
```

Comment:
You introduced an extra space :innocent: Probably doesn't matter for the rendered webpage, but still :joy: